data/i18n/pl/pl.toml

```diff
@@ -49,38 +49,35 @@ other = "CVE ID"
 [cve_issue_url]
 other = "URL do CVE w GitHubie"
 
-[cve_json_external_url]
-other = "external_url"
-
-[cve_json_id]
-other = "id"
-
-[cve_json_summary]
-other = "podsumowanie"
-
-[cve_json_url]
-other = "url"
-
 [cve_summary]
-other = "Podsumowanie problemu"
+other = "Streszczenie problemu"
 
 [cve_table]
 other = "Oficjalna lista CVE Kubernetesa"
 
-[cve_url]
-other = "CVE URL"
+[cve_table_date_format]
+other = "02 Jan 2006 15:04:05 MST"
+
+[cve_table_date_format_string]
+other = "(ostatnia modyfikacja: %s)"
 
 [deprecation_title]
-other = "Teraz oglądasz dokumentację Kubernetesa w wersji:"
+other = "Teraz czytasz dokumentację Kubernetesa dla wersji:"
 
 [deprecation_warning]
-other = " - dokumentacja nie jest już aktualizowana. Wyświetlona jest wersja archiwalna. Po aktualną dokumentację zajrzyj na"
+other = " ta dokumentacja nie jest już aktualizowana, a wersja, którą czytasz, ma charakter archiwalny. Najnowszą dokumentację znajdziesz tutaj "
 
 [deprecation_file_warning]
-other = "Przestarzały"
+other = "niewspierany"
+
+[outdated_content_title]
+other = "Informacje zawarte w tym dokumencie mogą nie być już aktualne"
+
+[outdated_content_message]
+other = "Ten dokument po raz ostatni został zmodyfikowany wcześniej niż wskazuje na to data publikacji jego wersji referencyjnej. To oznacza, że może być już nieaktualny. Jeśli czytasz po angielsku, zajrzyj do angielskiej, aktualizowanej na bieżąco, wersji dokumentacji: " 
```

Comment:
```suggestion
other = "Ten dokument po raz ostatni został zmodyfikowany wcześniej niż wskazuje na to data publikacji jego wersji referencyjnej. To oznacza, że może być już nieaktualny. Jeśli rozumiesz po angielsku, zajrzyj do aktualizowanej na bieżąco, angielskiej wersji dokumentacji: " 
```

Reply:
W oryginalne jest "if you're able to read English", dlatego proponowałbym zostawić wersję z "czytasz", też dlatego, że "rozumiesz" w tym kontekście mogłoby mieć negatywną, nieco paternalistyczną, konotację, jak np. "nie rozumiesz po angielsku?" albo "jeśli rozumiesz po angielsku, dlaczego nie korzystasz z wersji oryginalnej?" Inna sprawa, że sama wersja wyjściowa - moim zdaniem - mogłaby mieć nieco bardziej uprzejmy wydźwięk ;)

Reply:
Ja bym się tego nie obawiał, nigdy nie spotkałem się z rozumieniem wrażenia "_jeśli czytasz_" jako "_jeśli rozumiesz_".

Reply:
OK, to kompromisowo zmieniam na "znasz" ;)